code/modules/vehicles/unmanned/unmanned_droid.dm

```diff
@@ -9,6 +9,9 @@
 	gunnoise = 'sound/weapons/guns/fire/laser.ogg'
 	spawn_equipped_type = /obj/item/uav_turret/droid
 	unmanned_flags = HAS_LIGHTS|OVERLAY_TURRET
+	/// Reference to the datum used by the supply drop console
+	var/datum/supply_beacon/beacon_datum
+	var/datum/action/antenna/antenna = new
```

Comment:
missing autodoc and why isnt this new in Initialize

Reply:
It is extremly extremly unlikely (impossible) that this prevents compilation.

Declare the action here, but say it is equal to new in init